.github/workflows/ci.yaml

```diff
@@ -1,6 +1,11 @@
 name: CI
 
-on: push
+on:
+    pull_request:
+        types: [opened, ready_for_review, synchronize]
```

Comment:
JFYI `synchronize` is defined here:
https://github.com/orgs/community/discussions/24567#discussioncomment-3244510